support/nginx/peertube

```diff
@@ -37,6 +37,11 @@ server {
   # resolver $DNS-IP-1 $DNS-IP-2 valid=300s;
   # resolver_timeout 5s;
 
+  # Enable compression for JS/CSS/HTML and JSON, for improved client load times
+  gzip on;
+  gzip_types text/plain text/css text/html application/javascript application/json;
```

Comment:
Maybe we should disable gzip on `application/json`, to protect us from the [BREACH](http://breachattack.com/) attack.

Reply:
@Chocobozzz I think you are right, and that will make app faster, but one other solution could be good is to sign with hmac, that will totaly protect us from replay attack and modification.

Reply:
as I understand it, the issue here isn't necessarily about replay attacks (SSL itself protects us from that) but that when you compress then encrypt data, the size of the resulting packet can leak information about some of the encrypted plaintext if another part of the compressed and encrypted input was attacker-controlled.